yb-voyager/cmd/import.go

```diff
@@ -118,15 +118,17 @@ func registerImportDataFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&disablePb, "disable-pb", false,
 		"true - to disable progress bar during data import (default false)")
 	cmd.Flags().StringVar(&target.ExcludeTableList, "exclude-table-list", "",
-		"List of tables to exclude while importing data (no-op if --table-list is used)")
+		"list of tables to exclude while importing data (no-op if --table-list is used)")
 	cmd.Flags().StringVar(&target.TableList, "table-list", "",
-		"List of tables to include while importing data")
+		"list of tables to include while importing data")
 	cmd.Flags().Int64Var(&numLinesInASplit, "batch-size", DEFAULT_BATCH_SIZE,
-		"Maximum size of each batch import ")
+		"maximum size(in number of rows) of each batch to generate for import")
 	cmd.Flags().IntVar(&parallelImportJobs, "parallel-jobs", -1,
-		"Number of parallel copy command jobs. default: -1 means number of servers in the Yugabyte cluster")
+		"number of parallel copy command jobs to target database. "+
```

Comment:
number of parallel copy command jobs to provide to target database...
...will try to determine the total number of cores, N, and use N/2 as the parallel job count
[It falls back to using twice the number of nodes in the cluster **OR** It otherwise uses twice the number of nodes in the cluster]

Reply:
done

Reply:
done